rabbitmq/publisher.go

```diff
@@ -69,22 +82,19 @@ func (rp *rabbitMqPublisher) Run() {
 	}
 }
 
-// BroadcastChan returns a receive-only channel on which events are pushed by producers
-// Upon reading the channel, the hub publishes on the configured rabbitMQ channel
-func (rp *rabbitMqPublisher) BroadcastChan() chan<- data.BlockEvents {
-	return rp.broadcast
+// Broadcast will handle the block events pushed by producers, and sends them to rabbitMQ channel
+func (rp *rabbitMqPublisher) Broadcast(events data.BlockEvents) {
+	rp.broadcast <- events
```

Comment:
same suggestion with the closeChan already stated

Reply:
applied suggestion